utils/scripts/helpers/scan_daos_maldet.sh

```diff
@@ -48,7 +49,7 @@ sudo clamscan -d /usr/local/maldetect/sigs/rfxn.ndb    \
               --exclude-dir=/sys                       \
               --exclude-dir=/proc                      \
               --exclude-dir=/dev                       \
-              --exclude-dir=/scratch                   \
+              --exclude-dir=/CIShare                   \
```

Comment:
We still want to exclude /scratch here, and should also be excluding /mnt

Reply:
Done.